### ISSUE TITLE
announce typing notifications in instantbird

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4780]
**Reported by JamaicanUser on 2015-01-07 17:03**
In Instantbird, NVDA does not announce when the contact you are in a conversation with is typing. This causes me to type messages to users, only to get a response as I finish typing. Yes, the user can see if the person is typing with status line report, but this enhancement is a great need. Also, could shortcuts be added to report recent messages, and could we also hear messages from other convos reported outside the tab.

**Comment by leonardder** [comment]
It looks like Instantbird is quite abandoned, so it might be very unlikely that this will be implemented any time soon.

**Comment by Adriani90** [comment]
@JamaicanUser are you still using this software? Are you still having this issue?

**Comment by Adriani90** [comment]
I am closing this as won't fix since the application is dicontinued and we do not have any answer from the initial author since almost 4 years.
